recipes/ffmpeg/all/conanfile.py

```diff
@@ -902,11 +908,18 @@ def _add_component(name, dependencies):
             if Version(self.version) >= "5.0" and is_apple_os(self):
                 avfilter.frameworks.append("Metal")
 
+        if self.options.get_safe("with_libdrm"):
+            avutil.requires.append("libdrm::libdrm_libdrm")
         if self.options.get_safe("with_vaapi"):
-            avutil.requires.extend(["vaapi::vaapi", "xorg::x11"])
+            avutil.requires.append("vaapi::vaapi")
+            if self.options.get_safe("with_xcb"):
+                avutil.requires.append("xorg::x11")
```

Comment:
Are you sure about this change? What evidence could be used? At line 317 there is no dependency between vaapi and xcb

Reply:
It's been a while now, so I don't quite remember the exact details, but I remember this being an important dependency when enabling `vaapi`. It should be able to track down the dependency in the code for `avutil`.

Reply:
I mean, they were independent before. In case not having evidence, please, revert it. The FFMpeg is really critical, so you can open a new PR in the future, in case finding the reason with proper details.